server.js

```diff
@@ -1,7 +1,11 @@
-const BidiBot = require('./BidiBot');
-const config = require('./config');
+const BidiBot = require('./bidibot');
 const http = require('http');
 
+try {
```

Comment:
Should the app start if this config file doesn't exist? This seems like a way for the app to be silently failing.

Reply:
`config.js` handles the keys and secrets, which we instead pass via environment variables on Heroku.  Since there isn't a config.js there (adding one would pollute the git repo), it needs a bypass.  Probably should add a message to let the user know.